api/tasks.py

```diff
@@ -11,6 +11,8 @@ def create_delivery(order):
     for delivery in deliveries_by_vendor:
         if not delivery._is_errored():
             delivery.send_mail_to_vendor()
+        else:
+            delivery.send_error_mail_to_admin()
```

Comment:
IMO this is now the 3rd responsibility we are adding to the Delivery class, so it's the time to replace this with a new `FailedDeliveryMail`.

Reply:
Yes, i thought we agreed that next PR would be to refactor those into their own classes plus the django model persistance. Now is just a faster way to send the error by mail and have it in production if that's ok.

Reply:
but this is new code that we know will cause trouble. I don't see the reason why we need to add it and then fix it later. I see twice as work. Isn't it easier to do it right in the first place? is there something from my suggestion that you particularly difficult? I can provide more details if you want?

Reply:
No, it's all right, it was just a matter of speeding things and error debugging in production.

Reply:
I understand. In this case, there's no such rush considering this could take no more than 2h.

Reply:
I'll get to it today.

Reply:
Check the last refactor commit. Thanks.

Reply:
I meant only to refactor the email we're adding here, but refactoring both is far better :ok_hand: